script.js

```diff
@@ -1,92 +1,81 @@
 /* eslint-disable */
-// Obtener referencia a los elementos del DOM
 const form = document.querySelector('form');
-const titleInput = document.querySelector('input[placeholder="Title"]');
-const authorInput = document.querySelector('input[placeholder="Author"]');
-const listSection = document.querySelector('.list-section');
-
-// Obtener la colección de libros almacenada en localStorage o crear una nueva si no existe
-let booksCollection = JSON.parse(localStorage.getItem('booksCollection')) || [];
-
-// Agregar dos libros iniciales a la colección
-if (booksCollection.length === 0) {
-  const initialBooks = [
-    { title: 'Book 1', author: 'Author 1' },
-    { title: 'Book 2', author: 'Author 2' }
-  ];
-  booksCollection = initialBooks;
-  saveCollectionToLocalStorage();
+const bookList = document.getElementById('bookList');
+const menua = document.querySelector('.menua');
+const menub = document.querySelector('.menub');
+const menuc = document.querySelector('.menuc');
+const main = document.querySelector('.list');
+const add = document.querySelector('.add_new');
+const contact = document.querySelector('.contact');
+const books = [];
+
+class Abooks {
+  constructor(books) {
+    this.books = books;
+    this.local = () => {
+      if (localStorage.getItem('books')) {
+        books = JSON.parse(localStorage.getItem('books'));
+      }
+    };
+
+    this.displayBooks = () => {
+      if (localStorage.getItem('books')) {
```

Comment:
- [ ] Good job refactoring to classes from objects, however, the refactor should not break the code. A user should still be able to add and remove a book. Kindly fix your code.